src/arviz_plots/visuals/__init__.py

```diff
@@ -77,8 +84,12 @@ def ecdf_line(values, target, backend, **kwargs):
     return plot_backend.line(values.sel(plot_axis="x"), values.sel(plot_axis="y"), target, **kwargs)
 
 
-def fill_between_y(da, target, backend, *, x=None, y_bottom=None, y_top=None, **kwargs):
+def fill_between_y(da, target, backend, *, x=None, y_bottom=None, y=None, y_top=None, **kwargs):
     """Fill the region between to given y values."""
+    if y_bottom is None and y:
+        y_bottom = y
+    if y_top is None and y:
+        y_top = y
```

Comment:
I think I would modify this to make sure things behave as expected. `x` and `y` are mostly used so we can have aesthetics `x` and `y` that control shifting the visual elements within a plot. Therefore, I would expect that if I pass `y`, everything is shifted along the vertical axes. With these checks, if I give both `y_top` and `y` this won't happen.

I would use

```python
if y is not None:
    y_top += y
    y_bottom += y
```

once the `if "kwarg" in da.dims:` finishes as then both y_top and y_bottom should be array_like elements and not None.